framework_tvb/tvb/interfaces/web/controllers/spatial/region_stimulus_controller.py

```diff
@@ -133,16 +136,19 @@ def step_1(self):
         """
         current_stimuli_region = common.get_from_session(KEY_REGION_STIMULUS)
         selected_stimulus_gid = current_stimuli_region.gid.hex
-        region_stim_selector_form = StimulusRegionSelectorForm(common.get_current_project().id)
+        project_id = common.get_current_project().id
+        region_stim_selector_form = StimulusRegionSelectorForm(project_id)
         region_stim_selector_form.region_stimulus.data = selected_stimulus_gid
         region_stim_selector_form.display_name.data = common.get_from_session(KEY_REGION_STIMULUS_NAME)
 
-        region_stim_creator_form = RegionStimulusCreatorForm(self.equation_choices, common.get_current_project().id)
+        region_stim_creator_form = RegionStimulusCreatorForm(self.equation_choices, project_id)
         if not hasattr(current_stimuli_region, 'connectivity') or not current_stimuli_region.connectivity:
-            current_connectivity_in_form = region_stim_creator_form.connectivity._get_values_from_db()[0]
-            region_stim_creator_form.connectivity.data = current_connectivity_in_form[2]
```

Comment:
@popaula937 is it ok to remove this line, considering that we put the GID on the ViewModel, and then we do `form.fiil_from_trait` ?